boards.txt

```diff
@@ -2958,6 +2958,162 @@ S_ODI_Ultra.menu.EraseFlash.all.upload.erase_cmd=-e
+lilygo_t_display.menu.CPUFreq.13.build.f_cpu=13000000L
+lilygo_t_display.menu.CPUFreq.10=10MHz (40MHz XTAL)
+lilygo_t_display.menu.CPUFreq.10.build.f_cpu=10000000L
+
+lilygo_t_display.menu.FlashMode.qio=QIO
+lilygo_t_display.menu.FlashMode.qio.build.flash_mode=dio
+lilygo_t_display.menu.FlashMode.qio.build.boot=qio
+lilygo_t_display.menu.FlashMode.dio=DIO
+lilygo_t_display.menu.FlashMode.dio.build.flash_mode=dio
+lilygo_t_display.menu.FlashMode.dio.build.boot=dio
+lilygo_t_display.menu.FlashMode.qout=QOUT
+lilygo_t_display.menu.FlashMode.qout.build.flash_mode=dout
+lilygo_t_display.menu.FlashMode.qout.build.boot=qout
+lilygo_t_display.menu.FlashMode.dout=DOUT
+lilygo_t_display.menu.FlashMode.dout.build.flash_mode=dout
+lilygo_t_display.menu.FlashMode.dout.build.boot=dout
+
+lilygo_t_display.menu.FlashFreq.80=80MHz
+lilygo_t_display.menu.FlashFreq.80.build.flash_freq=80m
+lilygo_t_display.menu.FlashFreq.40=40MHz
+lilygo_t_display.menu.FlashFreq.40.build.flash_freq=40m
+
+lilygo_t_display.menu.FlashSize.4M=4MB (32Mb)
+lilygo_t_display.menu.FlashSize.4M.build.flash_size=4MB
+lilygo_t_display.menu.FlashSize.8M=8MB (64Mb)
+lilygo_t_display.menu.FlashSize.8M.build.flash_size=8MB
+lilygo_t_display.menu.FlashSize.8M.build.partitions=default_8MB
+lilygo_t_display.menu.FlashSize.2M=2MB (16Mb)
+lilygo_t_display.menu.FlashSize.2M.build.flash_size=2MB
+lilygo_t_display.menu.FlashSize.2M.build.partitions=minimal
+lilygo_t_display.menu.FlashSize.16M=16MB (128Mb)
+lilygo_t_display.menu.FlashSize.16M.build.flash_size=16MB
+
+lilygo_t_display.menu.UploadSpeed.921600=921600
+lilygo_t_display.menu.UploadSpeed.921600.upload.speed=921600
+lilygo_t_display.menu.UploadSpeed.115200=115200
+lilygo_t_display.menu.UploadSpeed.115200.upload.speed=115200
+lilygo_t_display.menu.UploadSpeed.256000.windows=256000
+lilygo_t_display.menu.UploadSpeed.256000.upload.speed=256000
+lilygo_t_display.menu.UploadSpeed.230400.windows.upload.speed=256000
+lilygo_t_display.menu.UploadSpeed.230400=230400
+lilygo_t_display.menu.UploadSpeed.230400.upload.speed=230400
+lilygo_t_display.menu.UploadSpeed.460800.linux=460800
+lilygo_t_display.menu.UploadSpeed.460800.macosx=460800
+lilygo_t_display.menu.UploadSpeed.460800.upload.speed=460800
+lilygo_t_display.menu.UploadSpeed.512000.windows=512000
+lilygo_t_display.menu.UploadSpeed.512000.upload.speed=512000
+
+lilygo_t_display.menu.DebugLevel.none=None
+lilygo_t_display.menu.DebugLevel.none.build.code_debug=0
+lilygo_t_display.menu.DebugLevel.error=Error
+lilygo_t_display.menu.DebugLevel.error.build.code_debug=1
+lilygo_t_display.menu.DebugLevel.warn=Warn
+lilygo_t_display.menu.DebugLevel.warn.build.code_debug=2
+lilygo_t_display.menu.DebugLevel.info=Info
+lilygo_t_display.menu.DebugLevel.info.build.code_debug=3
+lilygo_t_display.menu.DebugLevel.debug=Debug
+lilygo_t_display.menu.DebugLevel.debug.build.code_debug=4
+lilygo_t_display.menu.DebugLevel.verbose=Verbose
+lilygo_t_display.menu.DebugLevel.verbose.build.code_debug=5
+
```

Comment:
Please add EraseFlash at the end of board definition. Else this board will not work in Arduino IDE.
```
lilygo_t_display.menu.EraseFlash.none=Disabled
lilygo_t_display.menu.EraseFlash.none.upload.erase_cmd=
lilygo_t_display.menu.EraseFlash.all=Enabled
lilygo_t_display.menu.EraseFlash.all.upload.erase_cmd=-e
```